pkg/gatherers/conditional/conditional_gatherer_test.go

```diff
@@ -16,15 +17,15 @@ import (
 )
 
 func newEmptyGatherer() *Gatherer {
-	return &Gatherer{}
+	return New(nil, nil, "")
 }
 
 func Test_Gatherer_Basic(t *testing.T) {
 	gatherer := newEmptyGatherer()
 	assert.Equal(t, "conditional", gatherer.GetName())
 	gatheringFunctions, err := gatherer.GetGatheringFunctions(context.TODO())
-	assert.NoError(t, err)
-	assert.Greater(t, len(gatheringFunctions), 0)
+	assert.EqualError(t, err, `unable to load gathering rules: Get "": unsupported protocol scheme ""`)
```

Comment:
Maybe it would be slightly more realistic to pass some `gatheringRulesEndpoint` to the `New` function instead of an empty string.

Reply:
fixed